lib/udf_string_utils.rb

```diff
@@ -244,6 +244,55 @@ class UdfStringUtils
                            {query: "select ?(1, 3, 1, ',')", expect: "1,2" , example: true},
                            {query: "select ?(1, 5, 2, '|')", expect: "1|3", example: true},
                        ]
+      }, {
+          type:        :function,
```

Comment:
Thanks for the comments. I now created a new file called `udf_findhotel_utils.rb`. And defined only one function in there. I also added `UdfFindhotelUtils::UDFS,` line in `udf_harness.rb` file's `initialize` method. But when I try to run tests it always give the following error:

 `<class:UdfFindhotelUtils>': uninitialized constant UdfFindhotelUtils::None (NameError)`

Any suggestions?